### PR TITLE
GitHubTeacher adds Orlando, FL 

### LINKED ISSUE #20: YOUR-USERNAME follows GitHub Flow
# Orlando, FL

- [x] create a **branch**
- [x] add some **commits**
- [x] open a *pull request*
- [ ] collaborate
- [ ] `merge` with 👍

### PR DESCRIPTION
**Pull request by githubteacher**
This Pull Request is to add Orlando, FL to the list of hometowns. 

@crichID what kinds of things would you like to do with your son when you visit Orlando?

This PR closes #20.

### CODE REVIEW

**Comment by crichID** [comment]
> what kinds of things would you like to do with your son when you visit Orlando?

@githubteacher thank you for asking! My son loves trains and legos ... any recommendations?

**Comment by lamarr246** [comment]
Where is a good beach in Orlando?

**Comment by lamarr246** [comment]
Which is better for a quick visit--Universal Studios or Disney?

File: githubteacher-orlando.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Orlando, FL
+
+### Restaurants
+
+- The Pharmacy
+- Rocco's Tacos
+
```
Comment (lamarr246) [inline githubteacher-orlando.md]: Add Universal Studios

Comment (githubteacher) [inline githubteacher-orlando.md]: Done! It's been added in 16fe583

Comment (lamarr246) [inline githubteacher-orlando.md]: Looks great!!! 👍

**Comment by githubteacher** [comment]
> My son loves trains and legos ... any recommendations?

@crichID I added some recommendations!

@crichID @lamarr246 can I get an approval 👍 so I can merge?

**Pull request merged**